packages-next/admin-ui/src/templates/api.ts

```diff
@@ -0,0 +1,36 @@
+export const apiTemplate = `
```

Comment:
This code is a long way away from the very similar code in `createExpressServer`.

I wonder if there's a way we can bring those two things closer together? 🤔 

Also, since this isn't paramterised, should we put in in `static` and use `copy` operation rather than a `write`?

Reply:
> This code is a long way away from the very similar code in createExpressServer.

Hmm but there does seem to be a fairly heavy codependency between `admin-ui` and `system` it's already there as `createAdminUIServer` is imported into `createExpressServer` 

> Also, since this isn't paramterised, should we put in in static and use copy operation rather than a write?

Easily done!

Reply:
On second thought I think this should live in templates... I can't see a `static` dir in there but it's definitely a template even if it's not dynamic currently. I think moving it away from there might be a bit weird. 

Lemme know if I've misunderstood something.